Address double-cast in ReadValues

```
..\src\main.cc(161): warning C4244: 'initializing': conversion from 'double' to 'ULONG', possible loss of data [C:\projects\lean-mean-registry-client\build\registry.vcxproj]
..\src\main.cc(166): warning C4312: 'type cast': conversion from 'ULONG' to 'HKEY' of greater size [C:\projects\lean-mean-registry-client\build\registry.vcxproj]
     Creating library C:\projects\lean-mean-registry-client\build\Release\registry.lib and object C:\projects\lean-mean-registry-client\build\Release\registry.exp
```

This was some lazy code to get this up and running sooner - we can do better.

From the JS side, I've got a lookup that converts the string enum to the `long` specified in MSDN:

https://github.com/desktop/registry-js/blob/50dd82ef2ea9c93560c8d4ead14aa72f5bc51bda/lib/registry.ts#L70-L83

Then, in the C++ side, we need to cast this to a `HKEY` to use it with `RegOpenKeyEx`:

https://github.com/desktop/registry-js/blob/7b0da431a747b7093670e6a9b23b282707062bf7/src/main.cc#L152-L161

These look like this in `winreg.h`:

```c++
#define HKEY_CLASSES_ROOT                   (( HKEY ) (ULONG_PTR)((LONG)0x80000000) )
#define HKEY_CURRENT_USER                   (( HKEY ) (ULONG_PTR)((LONG)0x80000001) )
#define HKEY_LOCAL_MACHINE                  (( HKEY ) (ULONG_PTR)((LONG)0x80000002) )
#define HKEY_USERS                          (( HKEY ) (ULONG_PTR)((LONG)0x80000003) )
#define HKEY_PERFORMANCE_DATA               (( HKEY ) (ULONG_PTR)((LONG)0x80000004) )
#define HKEY_PERFORMANCE_TEXT               (( HKEY ) (ULONG_PTR)((LONG)0x80000050) )
#define HKEY_PERFORMANCE_NLSTEXT            (( HKEY ) (ULONG_PTR)((LONG)0x80000060) )
```

Just need to figure out how to marshal these properly from V8's types...

--- a/src/main.cc
+++ b/src/main.cc
@@ -158,12 +158,12 @@ NAN_METHOD(ReadValues)
     return;
   }
 
-  ULONG first = info[0]->NumberValue();
+  auto first = reinterpret_cast<HKEY>(info[0]->IntegerValue());
   auto second = *v8::String::Value(info[1]);
 
   HKEY hCurrentKey;
   LONG openKey = RegOpenKeyEx(
-    (HKEY)first,
+    first,
     (LPWSTR)second,
     0,
     KEY_READ | KEY_WOW64_64KEY,

